docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,183 @@
+description: Learn how to run accessibility tests using axe DevTools® Mobile Analyzer drivers hosted in Sauce Labs' private cloud.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><span className="sauceGreen">Real Devices</span></p>
+
+[Deque's axe™](https://www.deque.com/axe/) is a leading digital accessibility toolkit. 
+The [axe DevTools® for Mobile SDK](https://docs.deque.com/devtools-mobile/appium) extends this functionality to mobile platforms, 
+enabling automated accessibility testing directly in your Appium workflows for both Android and iOS apps. 
+This feature supports compliance with accessibility standards such as WCAG and the European Accessibility Act, 
+ensuring your apps are inclusive for all users.
+
+With this integration, you can:
+
++ Detect and address accessibility issues early in the development cycle.
++ Automate mobile accessibility testing using Sauce Labs' secure and scalable cloud infrastructure.
++ Gain actionable insights to improve compliance with global accessibility standards.
+
+The following guide explains how to set up Sauce Labs’ integration with [axe DevTools® for Mobile](https://docs.deque.com/devtools-mobile/appium) 
+and leverage its powerful tools to optimize accessibility testing in your mobile apps.
+
+
+
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+- Familiarity writing and running [Appium tests](/mobile-apps/automated-testing/appium/)
+- [A Deque Account](https://axe.deque.com/plans) (axe DevTools® for Mobile Product is required) 
+
+
+## Appium 2 Version: `appium2-deque-accessibility`
+
+Sauce Labs offers various versions of Appium 2 in our cloud. One of these is the `appium2-deque-accessibility` version, 
+designed to support accessibility testing on mobile apps for both iOS and Android. This version integrates Appium drivers developed by 
+the Deque team, enabling seamless integration of accessibility scans into your existing test automation workflows.
+
+You can find more details on Sauce Labs' [Appium versions documentation](/mobile-apps/automated-testing/appium/appium-versions/#appium-2x).
+
+## Key Features of `appium2-deque-accessibility`
+
+- ****Includes Deque Drivers :**** This version incorporates the following accessibility drivers provided by Deque:
+    - axe-appium-uiautomator2-driver: Automation name is `AxeUIAutomator2`
+    - axe-appium-xcuitest-driver: Automation name is `AxeXCUITEST`
+:::info Min iOS Version for `axe-appium-xcuitest-driver`:
+- You can use the `axe-appium-xcuitest-driver` only for devices with iOS 17 or above. 
+- To ensure compatibility, set the `appium:platformVersion` capability to `'^1(7|8).*$'` in your Appium configuration.
```

Comment:
I would not go into the specific here with `'^1(7|8).*$'` Its going to be outdated. I think we could just point to our doc for the platformVersion cap. The rest the customer needs to figure out based on what versions axe actually supports

Reply:
especially since this is only for iOS, if customers add this for android they will mess up, so we should let people go to axe doc

Reply:
Updated this, in my next commit